core/src/mindustry/ui/dialogs/ModsDialog.java

```diff
@@ -133,11 +144,121 @@ void setup(){
+                            BaseDialog dialog2 = new BaseDialog("$mod.featured.dialog.title");
+                            dialog2.cont.table(table -> {
+                                table.left();
+                                table.image(Icon.zoom);
+                                searchBar = table.field(searchtxt, res -> {
+                                    searchtxt = res;
+                                    rebuildBrowser[0].run();
+                                }).growX().get();
+                            }).fillX().padBottom(4);
+
+                            dialog2.cont.row();
+
+                            dialog2.cont.pane(tablebrow -> {
+                                tablebrow.margin(10f).top();
+                                rebuildBrowser[0] = () -> {
+                                    tablebrow.clear();
+                                    String searchString = searchtxt.toLowerCase();
+                                    Core.net.httpGet("https://raw.githubusercontent.com/Anuken/MindustryMods/master/mods.json", response -> {
+                                        if (response.getStatus() != HttpStatus.OK) {
+                                            ui.showErrorMessage(Core.bundle.format("connectfail", response.getStatus()));
+                                        } else {
+                                            Json json = new Json();
+                                            Seq<ModListing> listings = json.fromJson(Seq.class, ModListing.class, response.getResultAsString());
+                                            for (ModListing modsbrolist : listings) {
+                                                if((!searchtxt.isEmpty() && !modsbrolist.repo.contains(searchtxt))||searchtxt==null) continue;
+                                                tablebrow.button(btn -> {
+                                                    btn.top().left();
+                                                    btn.margin(12f);
+                                                    btn.table(con -> {
+                                                        con.left();
+                                                        con.add("[lightgray]Name:[] " + modsbrolist.name + "\n[lightgray]Author:[] " + modsbrolist.author + "\n[accent]\uE809: " + modsbrolist.stars).width(388f).wrap().growX().pad(0f,6f,0f,6f);
+                                                        con.add().growX().pad(0f,6f,0f,6f);
+                                                    }).fillY().growX().pad(0f,6f,0f,6f);
+                                                }, Styles.clearPartialt, () -> {
+                                                    BaseDialog modSelected = new BaseDialog((Core.bundle.get("mods.browser.selected"))+ ": "+ modsbrolist.name);
+                                                    modSelected.cont.add(modsbrolist.description).width(mobile ? 400f : 500f).wrap().pad(4f).get().setAlignment(Align.center, Align.center);
```

Comment:
Don't display the description in the mod list. It should only be visible when clicking the mod.

Reply:
It _is_ only shown when clicked

Reply:
My bad, I didn't see the more recent preview